src/test/iv_client.c

```diff
@@ -65,13 +65,6 @@ static crt_endpoint_t	g_server_ep;
 static char		g_hostname[100];
 static bool		g_do_shutdown;
 
-
-#define DBG_PRINT(x...)					\
```

Comment:
minor note - new debug print macros are not quite the same as they do not print the hostname